src/main/java/gregtech/api/util/GTUtility.java

```diff
@@ -1179,6 +1184,25 @@ public static int computeLatchedRedstoneBetweenValues(float value, float maxValu
         return output;
     }
 
+    /**
+     * Compares a current against max, with an option to invert the logic.
+     *
+     * @param current    value to be compared
+     * @param max        the max that value can be
+     * @param isInverted whether to invert the logic of this method
+     * @return value 0 to 15
+     * @throws ArithmeticException when max is 0
+     */
+    public static int computeRedstoneValue(long current, long max, boolean isInverted) throws ArithmeticException {
```

Comment:
Could this method, and the above `computeLatchedRedstoneBetweenValues` go in a new GTRedstoneUtility class?

Reply:
Moved